pkg/controller/user-cluster-controller-manager/resources/resources/mla/prometheus/configmap.go

```diff
@@ -0,0 +1,321 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package prometheus
+
+import (
+	"bytes"
+	"text/template"
+
+	"k8c.io/kubermatic/v2/pkg/resources"
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+type Config struct {
+	MLAGatewayURL string
+}
+
+func ConfigMapCreator(config Config) reconciling.NamedConfigMapCreatorGetter {
+	return func() (string, reconciling.ConfigMapCreator) {
+		return resources.UserClusterPrometheusConfigMapName, func(configMap *corev1.ConfigMap) (*corev1.ConfigMap, error) {
+			if configMap.Data == nil {
+				configMap.Data = map[string]string{}
+			}
+			t, err := template.New("prometheus").Parse(configTemplate)
+			if err != nil {
+				return nil, err
+			}
+			configBuf := bytes.Buffer{}
+			if err := t.Execute(&configBuf, config); err != nil {
+				return nil, err
+			}
+			configMap.Data["prometheus.yaml"] = configBuf.String()
+			configMap.Labels = resources.BaseAppLabels(resources.UserClusterPrometheusConfigMapName, nil)
+			return configMap, nil
+		}
+	}
+}
+
+const (
+	configTemplate = `
+global:
+  evaluation_interval: 1m
+  scrape_interval: 1m
+  scrape_timeout: 10s
+remote_write:
+- url: {{ .MLAGatewayURL }}
+rule_files:
+- /etc/config/recording_rules.yml
+- /etc/config/alerting_rules.yml
+- /etc/config/rules
+- /etc/config/alerts
+scrape_configs:
+- job_name: prometheus
+  static_configs:
+  - targets:
+    - localhost:9090
+- bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  job_name: kubernetes-apiservers
+  kubernetes_sd_configs:
+  - role: endpoints
+  relabel_configs:
+  - action: keep
+    regex: default;kubernetes;https
+    source_labels:
+    - __meta_kubernetes_namespace
+    - __meta_kubernetes_service_name
+    - __meta_kubernetes_endpoint_port_name
+  scheme: https
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
+- bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  job_name: kubernetes-nodes
+  kubernetes_sd_configs:
+  - role: node
+  relabel_configs:
+  - action: labelmap
+    regex: __meta_kubernetes_node_label_(.+)
+  - replacement: kubernetes.default.svc:443
+    target_label: __address__
+  - regex: (.+)
+    replacement: /api/v1/nodes/$1/proxy/metrics
+    source_labels:
+    - __meta_kubernetes_node_name
+    target_label: __metrics_path__
+  scheme: https
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
```

Comment:
why true (here, and below)?

Reply:
As mentioned above, mla in Seed is not ready, is that ok for you we handle all tls config later? We have an issue about enable mTLS auth for Gateway: https://github.com/kubermatic/kubermatic/issues/6493